wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -231,11 +243,24 @@ public Response create(
           @DefaultValue("false")
           Boolean startAfterCreate,
       @ApiParam("Namespace where workspace should be created") @QueryParam("namespace")
-          String namespace)
+          String namespace,
+      @Context HttpHeaders headers)
       throws ConflictException, BadRequestException, ForbiddenException, NotFoundException,
           ServerException {
     requiredNotNull(devfile, "Devfile");
 
+    DevfileImpl devfileModel;
```

Comment:
Would it be simple to get this method parameter ```@HeaderParam("Content-Type") MediaType type```
and compare ``` if(type.equals(MediaType.APPLICATION_JSON)){}else {}```?

Reply:
It would :)